pkgs/games/rare/default.nix

```diff
@@ -3,6 +3,7 @@
 , buildPythonApplication
 , qt5
 , legendary-gl
+, orjson
```

Comment:
Quick question, why is this dependency being added? I didn't see it on the `setup.py`.